CHANGELOG.md

```diff
@@ -28,6 +29,7 @@
 ### Snowpark Local Testing Updates
 
 #### New Features
+
 - Added support for the following APIs:
   - snowflake.snowpark.functions
     - rank
```

Comment:
Put backticks around "rank" because it's a function name.

Reply:
If there's only one function in the list, maybe just use a sentence? For example:

```
Added support for the `snowflake.snowpark.functions.rank` API.
 ```